DependencyInjection/Configuration.php

```diff
@@ -48,6 +48,30 @@ public function getConfigTreeBuilder()
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('orm')
+                            ->addDefaultsIfNotSet()
+                            ->canBeEnabled()
+                            ->children()
+                                ->scalarNode('manager_registry')->defaultValue('doctrine')->end()
+                                ->scalarNode('manager_name')->defaultNull()->end()
+                                ->scalarNode('entity_class')->defaultValue('Symfony\Cmf\Bundle\SimpleCmsBundle\Doctrine\Orm\Page')->end()
+
+                                /*
+                                ->enumNode('use_sonata_admin')
+                                    ->values(array(true, false, 'auto'))
+                                    ->defaultValue('auto')
+                                ->end()
+                                ->arrayNode('sonata_admin')
+                                    ->children()
+                                        ->enumNode('sort')
+                                            ->values(array(false, 'asc', 'desc'))
+                                            ->defaultValue(false)
+                                        ->end()
+                                    ->end()
+                                ->end()
+                                */
```

Comment:
we need to either implement sonata or remove these commented out configuration parts